config/locales/en.yml

```
@@ -248,6 +251,12 @@ en:
     not_found_agent: Agent with id %{id}
     not_used: Not used
     ontology_not_valid: 'ontology is not valid, here are the errors: '
+    profile:
+      contributed_in: Has a role in
+      metadata: Metadata
+      statistics: Statistics
+      no_contributions: This agent has no contributions in %{portalName} 
+      keywords : Keywords
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)